src/test/infrastructure/nfs-server/templates/statefulset.yaml

```diff
@@ -125,7 +125,7 @@ spec:
         {{- end }}
       spec:
         accessModes: [ "ReadWriteOnce" ]
-        storageClassName: {{ .Values.persistence.storageClassName | default "gp2" }}
+        storageClassName: {{ .Values.persistence.storageClassName }}
```

Comment:
What about not including the `storageClassName`property in case the `.Values.peristance.storageClassName` is empty? I assume it should then use the default storage class:

> If storageClassName is not specified in the PVC, the default storage class will be used for provisioning.

https://kubernetes.io/blog/2017/03/dynamic-provisioning-and-storage-classes-kubernetes/

Reply:
Yes, I think this is a better idea. To be honest, I wonder why it works now, because the empty string is supposed to disable the dynamic provisioning, according to the documentation.

Reply:
Maybe you want to do something like this:
`  {{- if not (kindIs "invalid" .Values.volumes.sharedHome.persistentVolumeClaim.storageClassName) }}
  storageClassName: {{ .Values.volumes.sharedHome.persistentVolumeClaim.storageClassName | quote }}
  {{- end }}`
(from src/main/charts/bitbucket/templates/shared-home-pvc.yaml)

Reply:
Thanks for the tip. I've done this and added a comment to explain the usage.